Jenkinsfile

```diff
@@ -94,6 +94,23 @@ pipeline {
             }
         }
 
+        stage('Run CFT tests') {
+            options {
+                timeout(time: 10, unit: 'MINUTES')
+            }
+            steps {
+                sh 'tests/test_crash_fault_tolerance.sh'
```

Comment:
The Jenkins tests should be run using `bin/run_docker_test` (see https://github.com/hyperledger/sawtooth-raft/blob/master/Jenkinsfile)